index.js

```diff
@@ -10,60 +10,59 @@ const isValidUrl = (urlString) => {
     return false;
   }
 };
-
 const url = process.argv[process.argv.length - 1];
-if (isValidUrl(url)) {
-  // Get all network requests of a page
-  const browser = await puppeteer.launch();
-  const page = await browser.newPage();
-  await page.setRequestInterception(true);
-  const requests = [];
-  page.on("request", (request) => {
-    requests.push(request.url());
-    request.continue();
-  });
-  await page.goto(url, {
-    waitUntil: "networkidle2",
-  });
-  await browser.close();
+if (!isValidUrl(url)) {
+  console.log("Run this script with the url as the last parameter");
+  return;
+}
+// Get all network requests of a page
+const browser = await puppeteer.launch();
+const page = await browser.newPage();
+await page.setRequestInterception(true);
+const requests = [];
+page.on("request", (request) => {
+  requests.push(request.url());
+  request.continue();
+});
+await page.goto(url, {
+  waitUntil: "networkidle2",
+});
+await browser.close();
+
+// Get sets and icons from requests
+const iconifyRequests = requests.filter((item) =>
+  item.startsWith("https://api.iconify.design/"),
+);
 
-  // Get sets and icons from requests
-  const iconifyRequests = requests.filter((item) =>
-    item.startsWith("https://api.iconify.design/")
+if (iconifyRequests.length === 0) {
+  console.log("Looks like you don't have any iconify icons on this page");
+  return;
```

Comment:
There isn't a function wrapping all of this, so these returns won't work `process.exit()` would work instead

Reply:
Replaced them with `process.exit(1)` (as technically those are errors / bad outcomes) in https://github.com/samrobbins85/iconify-license-checker/pull/1/commits/70d634196a51394dbdcff74ac38762aac751d3b6